src/index.js

```diff
@@ -1,44 +1,20 @@
 import './style.css';
 
-class Score {
-  constructor(name, score) {
-    this.name = name;
-    this.score = score;
-  }
-}
+import { SubmitData, DispalyScore } from './submit-display.js';
 
-const DispalyScore = () => {
-  const existScore = JSON.parse(localStorage.getItem('scores'));
-  if (existScore !== null && existScore.length > 0) {
-    const scoreList = document.querySelector('.score-list');
-    const scoreItem = document.createElement('ul');
-    scoreList.innerHTML = '';
+const refreshBtn = document.querySelector('.refresh-btn');
+const submitForm = document.querySelector('.add-score');
 
-    existScore.forEach((score) => {
-      const scoreListItem = document.createElement('li');
-      scoreListItem.innerHTML = `${score.name}  :  ${score.score}`;
-      scoreItem.appendChild(scoreListItem);
-      scoreList.appendChild(scoreItem);
-    });
-  }
-};
-
-const storeScore = (e) => {
+submitForm.addEventListener('submit', (e) => {
   e.preventDefault();
-  let existScore = JSON.parse(localStorage.getItem('scores'));
-  existScore = existScore == null ? [] : existScore;
-  const inputName = document.querySelector('.name').value;
-  const inputScore = document.querySelector('.score').value;
+  SubmitData();
+});
 
-  const newScore = new Score(inputName, inputScore);
+// submitBtn.addEventListener('click', (e) => {
+//   e.preventDefault();
+//   SubmitData();
+// });
```

Comment:
- [ ] it would be better if you remove the unused commented code in order to keep you file clean and pro